C/sort/quick_sort.c

```diff
@@ -0,0 +1,68 @@
+#include <stdio.h>
```

Comment:
Your code is copied from [here](https://www.javatpoint.com/quick-sort)
We take plagiarism very seriously. Only original contribution will be reviewed and accepted.